build-app-tests-container.sh

```diff
@@ -10,7 +10,7 @@ FLAVOR=debug
 if [ -z "$SKIP_RUNNING_TESTS" ]; then
   make tests-container FLAVOR=$FLAVOR
   TESTS_CONTAINER_TAG=$(cat build/nitro-$FLAVOR/tests-container-tag)
-  TESTS_CONTAINER_ECR="513076507034.dkr.ecr.us-west-1.amazonaws.com/development-images/$TESTS_CONTAINER_TAG"
+  TESTS_CONTAINER_ECR="513076507034.dkr.ecr.us-west-1.amazonaws.com/salmiac-github-ci/$TESTS_CONTAINER_TAG"
```

Comment:
Do we use this script in salmiac-release job or a different script?

Can we ensure that github artifacts are not used by the salmiac-release job?

Reply:
I don't think we use it anywhere else